### PR TITLE
Adding README

### PR DESCRIPTION
**Pull request by gildedgardenia**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+## Assignment Requirements
+
+For this "Launch Checklist Form" assignment, you will use a form submission and fetch request to accomplish the following:
+
+1. Validate the user responses with `preventDefault()` to ensure the following: 
+
+   a. The user has entered something for every field.
+   b. The user has entered string values for names and number for fuel and cargo levels.
+
+1. With validation, update a list of what is currently ready or not ready for the shuttle launch.
```
Comment (codinglikeagirl42) [inline README.md]: Formatting is off when previewing file -> 

a and b should be on separate lines

File: README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+1. On valid form submission:
+
+   a. Do the pilot and co-pilot values submitted appear as the names in the third box?
+   b. Does the fuel value message reflect if the submitted value is too low (less than 10,000) or in the correct range?
+   c. Does the cargo value message reflect if the submitted value is too high (more than 10,000) or in the correct range?
+   d. Is the header message red and does it read 'Shuttle Not Ready for Launch' when either fuel or cargo are not in range?
+   e. Is the header message green and does it read 'Shuttle is Ready for Launch' when both fuel and cargo are submitted in range?
+
+1. You added JavaScript in ``script.js`` that is enclosed within a load event listener.
+1. A submit event listener handles the form submission.
```
Comment (codinglikeagirl42) [inline README.md]: Formatting is off when previewing the file.

The second ordered list elements are not on separate lines:
a
b

But all on one line.

ab

**Review by codinglikeagirl42 — CHANGES_REQUESTED** [review]
There should only be 1 file changed. Please add the node_modules to .gitignore and recommit.

File: README.md (hunk 1 above)
Comment (gildedgardenia) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (gildedgardenia) [reply README.md]: Done!

**Comment by gildedgardenia** [comment]
> There should only be 1 file changed. Please add the node_modules to .gitignore and recommit.

We can't add node_modules to .gitignore without breaking the autograder and making the students' lives more difficult.

**Review by codinglikeagirl42 — APPROVED** [review]

**Pull request merged**